configurator/src/main/java/com/devonfw/tools/ide/migrator/Migrations.java

```diff
@@ -147,6 +147,28 @@ public static MigrationImpl devon4j() {
         .pom().replaceProperty("devon4j.version", "3.1.1") //
         .replaceProperty("jackson.version", "2.9.9.20190727") //
         .and() //
+        .next().to(VersionIdentifier.ofDevon4j("3.2.0")) //
+        .pom().replaceProperty("devon4j.version", "3.2.0") //
+        .replaceString("com.devonfw.module.test.common.api.category.CategorySystemTest", "system")
+        .replaceString(
+            "com.devonfw.module.test.common.api.category.CategoryComponentTest,com.devonfw.module.test.common.api.category.CategorySubsystemTest,com.devonfw.module.test.common.api.category.CategorySystemTest",
+            "component,subsystem,system")
+        .replaceString(
+            "com.devonfw.module.test.common.api.category.CategorySubsystemTest,com.devonfw.module.test.common.api.category.CategorySystemTest",
+            "subsystem,system")
+        .replaceDependency(new VersionIdentifier("junit", "junit", null),
+            new VersionIdentifier("org.junit.jupiter", "junit-jupiter-engine", null))
+        .addDependency(new VersionIdentifier("*-core", null),
+            new VersionIdentifier("org.junit.platform", "junit-platform-runner", null))
```

Comment:
Github review seems buggy. Comment was lost, so here again:
We need to add this dependency with scope test. However, this is not yet possible. The esiest way would IMHO be to add `scope` as String property to `VersionIdentifier` with getter/setter and further constructor.
Then in https://github.com/devonfw/ide/blob/2846c26e13230dc4d8ac51a61ef44d14a6f12635/configurator/src/main/java/com/devonfw/tools/ide/migrator/xml/MavenDependencyAdder.java#L61 we could add something like this:
```
        String scope = this.dependency.getScope();
        if (scope != null) {
          Element dependencyScopeElement = xml.createElement("scope");
          dependencyScopeElement.setTextContent(scope);
          dependencyElement.appendChild(dependencyScopeElement);
        }
```

Reply:
done